File: Platron/Pages/BasePage.cs
using System;
using OpenQA.Selenium;
using Platron.Core.Service;
using SeleniumExtras.PageObjects;

namespace Platron.Pages
{
    public class BasePage
    {
        private readonly IWebDriver driver;
        protected BasePage()
        {
            this.driver = ServiceDriver.Driver;
            PageFactory.InitElements(driver, this);
        }

        [FindsBy(How = How.LinkText, Using = "Войти")]
        public IWebElement ButtonEnter { get; set; }
    }
}

File: Platron/src/Pages/BasePage.java
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.support.FindBy;
import org.openqa.selenium.support.PageFactory;

public class BasePage {
    private final WebDriver driver;
    @FindBy(linkText = "Войти")
    public WebElement ButtonEnter;

    protected BasePage()
    {
        this.driver = ServiceDriver.Driver;
        PageFactory.initElements(driver, this);
    }
}
